lib/document.js

```diff
@@ -3897,6 +3899,24 @@ Document.prototype.populate = function populate() {
   return this;
 };
 
+/* Returns an array of all populated documents associated with the query. */
+Document.prototype.$getPopulatedDocs = function $getPopulatedDocs() {
+  const index = (Object.keys(this.$__.populated));
```

Comment:
Let's rename this variable to `keys` or something that's more indicative of how this is used.